tempest2/src/main/kotlin/app/cash/tempest2/WritingPager.kt

```diff
@@ -49,33 +49,35 @@ class WritingPager<T> @JvmOverloads constructor(
     val currentPageSize = handler.beforePage(remainingUpdates, maxTransactionItems)
     val currentPage = remainingUpdates.take(currentPageSize)
 
-    val writeSet = TransactionWriteSet.Builder()
+    val builder = TransactionWriteSet.Builder()
     val appliedUpdates = mutableListOf<T>()
 
     while (appliedUpdates.size < currentPage.size &&
-      writeSet.size <= maxTransactionItems
+      builder.size <= maxTransactionItems
     ) {
       val newEntity = currentPage[appliedUpdates.size]
 
       val itemWriteSet = TransactionWriteSet.Builder()
       handler.item(itemWriteSet, newEntity)
 
-      if (writeSet.size + itemWriteSet.size > maxTransactionItems) {
+      if (builder.size + itemWriteSet.size > maxTransactionItems) {
         break // This item would have caused us to exceed the page limit. Skip it.
       }
 
-      writeSet.addAll(itemWriteSet)
+      builder.addAll(itemWriteSet)
       appliedUpdates += newEntity
     }
 
     if (appliedUpdates.isEmpty()) {
       return 0 // Not updating any items. Discard the transaction.
     }
 
-    handler.finishPage(writeSet)
-    check(writeSet.size <= maxTransactionItems) { "finishPage wrote too many items" }
+    handler.finishPage(builder)
+    check(builder.size <= maxTransactionItems) { "finishPage wrote too many items" }
 
-    db.transactionWrite(writeSet.build())
+    val writeSet = builder.build()
```

Comment:
nit: I think the rename loses something, and it looks like it was done to use the name for the built version. Feels like a shame that the name used more is the less clear one. 

How do you feel about `val [transaction | page] = writeSet.build()` ?

Reply:
`page` SGTM. done.